deps/docker-compose.yml

```diff
@@ -21,10 +22,28 @@ services:
     ports:
       - 5432
 
+  pgadmin:
```

Comment:
please remove everything related to pgadmin from this docker-compose. pgadmin has already been added to the `docker-compose-tools.yml` in this folder. We should not mix tools and dependencies.